public/javascripts/SVValidate/src/label/LabelVisibilityControl.js

```diff
@@ -129,5 +163,4 @@ function LabelVisibilityControl () {
     self.hideTagsAndDeleteButton = hideTagsAndDeleteButton;
 
     return this;
-}
```

Comment:
And according to our [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide), we want to have a newline at the end of files. It looks like maybe your IDE is automatically changing files you edit to remove newlines. Can you change the settings to do the opposite for this project?